### PR TITLE
pass user roles with login_success

### PR DESCRIPTION
**Pull request by harunurhan**
* Passes json serilized user role names for the current user to the
opener tab (inspirehep-ui)

### CODE REVIEW

File: inspirehep/modules/theme/views.py [hunk 1]
```diff
@@ -806,7 +806,8 @@ def login_success():
         'inspirehep_theme/login_success.html',
         user={
             'data': {
-                'email': current_user.email
+                'email': current_user.email,
+                'roles': map(lambda role: role.name, current_user.roles)
```
Comment (michamos) [inline inspirehep/modules/theme/views.py]: minor: list comprehensions are more Pythonic `[role.name for role in current_user.roles]`.

Comment (harunurhan) [reply inspirehep/modules/theme/views.py]: I always forget existence of list comprehensions 😄

**Comment by harunurhan** [comment]
ci merge please

**Comment by harunurhan** [comment]
ci merge please

**Pull request merged**